libs/wingsdk/test/target-sim/container.test.ts

```diff
@@ -43,6 +43,7 @@ test("simple container from registry", async () => {
 });
 
 test("simple container from a dockerfile", async () => {
+  // TODO: fix this test
```

Comment:
Should this be fixed before merging this PR?

Reply:
Probably. Damn

Reply:
My bad I already fixed it but forgot to remove the comment